packages/snaps-types/src/types.ts

```diff
@@ -7,6 +7,8 @@ export type Ethereum = StreamProvider;
 
 // Exported again for convenience.
 export type { Json, JsonRpcRequest } from '@metamask/utils';
+export { DialogType, NotificationType } from '@metamask/rpc-methods';
```

Comment:
Why some imports are `type` and some aren't?

Reply:
Enums are both types and runtime values. Using a type export here would make it impossible to use the enum value.